centreon/tests/e2e/cypress/e2e/Dashboards/02-dashboard-navigation.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Navigate through the list of dashboards
+  As a Centreon user with appropriate rights 
+  I need to navigate through the list of available dashboards 
+  In order to locate the one I want to display or edit
+
+Scenario: Get on the listing page when no dashboards are available  
+  Given a user with access to the dashboards library
+  When they access the dashboard listing page on a platform with no dashboards
+  Then a special message and a button to create a new dashboard are displayed instead of the dashboards
```

Comment:
```suggestion
  Then a special message and a button to create a new dashboard are displayed instead of the dashboards
```
```suggestion
  Then a message and a button to create a new dashboard are displayed instead of the dashboards
```